File: java/logica_y_validaciones/validar_numero_valido.java
package logica_y_validaciones;
public class validar_numero_valido {
    public static void main(String[] args) {
        String telefono = "3001234567";
        if (telefono.length() == 10) {
            System.out.println("Número válido.");
        } else {
            System.out.println("Número inválido.");
        }
    }
}

File: visual_basic/condicionales/Validar número válido/validar_numero_valido.vb
Imports System
Module validar_numero_valido
    Sub Main(args As String())
        Dim entrada As String
        Dim numero As Integer
        Console.Write("Ingrese un número: ")
        entrada = Console.ReadLine()
        If Integer.TryParse(entrada, numero) Then
            Console.WriteLine("Número válido: " & numero)
        Else
            Console.WriteLine("Error: No ingresaste un número válido")
        End If
    End Sub
End Module